src/attack_flow_builder/src/assets/builder.config.publisher.ts

```diff
@@ -259,7 +259,16 @@ class AttackFlowPublisher extends DiagramPublisher {
                 case PropertyType.Enum:
                     if (prop instanceof EnumProperty && prop.isDefined()) {
                         let value = prop.toReferenceValue()!.toRawValue()!;
```

Comment:
To simplify this, just export the identifier of the selected enum (instead of trying to convert the enum's text to the identifier).

Replace let value = `prop.toReferenceValue()!.toRawValue()!;` with `let value = prop.toRawValue()!;`

In the `// case(BoolEnum)` use lowercase `true` and `false`.

In the `// case(String | List | Dictionary | null)` use `value` outright.